### ISSUE TITLE
[Issue]: Jellyfin 10.9.1 locking up

### ISSUE THREAD

**Issue by ShakeSp33r** [jellyfin/jellyfin#11624]
### Please describe your bug

I'm running the linuxserver/jellyfin:10.9.1 container. everything working fine and then Jellyfin locks up, everything freeze, cant connect via web or Android TV. I have to restart the container.

Had a look at the log file, and this was the last bit before having to restart the container.

### Reproduction Steps

Server is scanning chapters, and I have 4 users connected.

Locking up happens randomly.

### Jellyfin Version

10.9.0

### if other:

10.9.1

### Environment

```markdown
- Docker : linuxserver/jellyfin:10.9.1
- Clients: Android TV
- Reverse Proxy: Nginx
- No transcoding all direct play
```

### Jellyfin logs

```shell
[2024-05-13 21:32:00.057 +02:00] [INF] [51] Jellyfin.Plugin.PlaybackReporting.EventMonitorEntryPoint: Processing playback tracker : "b3e52a6b46e3f3ee98055a3ef1373b836f6cefd8-07e4f4f654074d04b991c825bf219e18-b20495b8de9b75181322a26bcd40801a"
[2024-05-13 21:32:26.670 +02:00] [INF] [58] Emby.Server.Implementations.Session.SessionWebSocketListener: Sending ForceKeepAlive message to 1 inactive WebSockets.
[2024-05-13 21:32:26.670 +02:00] [INF] [58] Emby.Server.Implementations.Session.SessionWebSocketListener: Sending ForceKeepAlive message to 1 inactive WebSockets.
[2024-05-13 21:33:02.751 +02:00] [INF] [100] Emby.Server.Implementations.Session.SessionWebSocketListener: Lost 4 WebSockets.
[2024-05-13 21:39:39.809 +02:00] [INF] [518] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Android TV" "0.16.10" playing "<<<movie>>>>". Stopped at "5581614" ms
[2024-05-13 21:41:20.808 +02:00] [INF] [628] Emby.Server.Implementations.Session.SessionManager: Sending shutdown notifications
[2024-05-13 21:41:27.322 +02:00] [ERR] [634] Jellyfin.Api.Middleware.ExceptionMiddleware: Error processing request: "Unexpected end of request content". URL "POST" "/Users/AuthenticateByName".
[2024-05-13 21:41:29.348 +02:00] [ERR] [636] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "xxx.xxx.xxx.xxx" WebSocketRequestHandler error

System.Net.WebSockets.WebSocketException (0x80004005): The WebSocket is in an invalid state ('Closed') for this operation. Valid states are: 'Open, CloseReceived'
   at System.Net.WebSockets.ManagedWebSocket.SendAsync(ReadOnlyMemory`1 buffer, WebSocketMessageType messageType, WebSocketMessageFlags messageFlags, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ProcessInternal(PipeReader reader)
   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ReceiveAsync(CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)

[2024-05-13 21:41:29.450 +02:00] [ERR] [636] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "xxx.xxx.xxx.xxx" WebSocketRequestHandler error
System.Net.WebSockets.WebSocketException (0x80004005): The WebSocket is in an invalid state ('Closed') for this operation. Valid states are: 'Open, CloseReceived'
   at System.Net.WebSockets.ManagedWebSocket.SendAsync(ReadOnlyMemory`1 buffer, WebSocketMessageType messageType, WebSocketMessageFlags messageFlags, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ProcessInternal(PipeReader reader)
   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ReceiveAsync(CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)

[2024-05-13 21:41:29.450 +02:00] [ERR] [636] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "xxx.xxx.xxx.xxx" WebSocketRequestHandler error
System.Net.WebSockets.WebSocketException (0x80004005): The WebSocket is in an invalid state ('Closed') for this operation. Valid states are: 'Open, CloseReceived'
   at System.Net.WebSockets.ManagedWebSocket.SendAsync(ReadOnlyMemory`1 buffer, WebSocketMessageType messageType, WebSocketMessageFlags messageFlags, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ProcessInternal(PipeReader reader)
   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ReceiveAsync(CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)

[2024-05-13 21:41:29.451 +02:00] [ERR] [636] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "xxx.xxx.xxx.xxx" WebSocketRequestHandler error
System.Net.WebSockets.WebSocketException (0x80004005): The WebSocket is in an invalid state ('Closed') for this operation. Valid states are: 'Open, CloseReceived'
   at System.Net.WebSockets.ManagedWebSocket.SendAsync(ReadOnlyMemory`1 buffer, WebSocketMessageType messageType, WebSocketMessageFlags messageFlags, CancellationToken cancellationToken)
--- End of stack trace from previous location ---

   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ProcessInternal(PipeReader reader)
   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ReceiveAsync(CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by felix920506** [comment]
What is the host operating system

**Comment by ShakeSp33r** [comment]
Debian GNU/Linux 11 (bullseye)

**Comment by JamsRepos** [comment]
I had a similar situation where the server looked fine in the logs but you could not connect to the front-end at all and this happened twice within 24 hours. I restarted the container and have been fine since, awaiting it to happen again to check the logs further, could be related, could be a fluke.

I'm using Unraid official docker image.

**Comment by felix920506** [comment]
@JamsRepos JF locking up on Unraid is a known issue and is being tracked in #11459

**Comment by l7ssha** [comment]
I have similar problem with websocket closing i presume. I dont know if thats the culprit since jellyfin hangs randomly, disconnects from client and then after some time or after restarting service it comes back again.
```
[2024-05-14 11:29:47.234 +02:00] [INF] User policy for "szymon". EnableAudioPlaybackTranscoding: True
[2024-05-14 11:29:47.234 +02:00] [INF] RemoteClientBitrateLimit: 30000000, RemoteIP: "172.18.0.1", IsInLocalNetwork: False
[2024-05-14 11:29:49.271 +02:00] [INF] Processing playback tracker : "f95d9848bfdddc3a4c500e6781ada5d23a6739e5-bc8e852f0187456c87b3dca5ed6cca4c-307e2fea2e166e6a347b1d6438a17b11"
[2024-05-14 11:29:49.914 +02:00] [INF] Processing playback tracker : "Feishin-Desktop Client-szymon-7b8e69c4c8064f969ac2cca8fb075d29-585978acdf41e1aafa677b17d9c857e3"
[2024-05-14 11:30:47.660 +02:00] [INF] Sending ForceKeepAlive message to 1 inactive WebSockets.
[2024-05-14 11:30:59.672 +02:00] [INF] Lost 1 WebSockets.
[2024-05-14 11:34:16.810 +02:00] [INF] Sending shutdown notifications
[2024-05-14 11:34:19.336 +02:00] [ERR] WS "192.168.0.207" WebSocketRequestHandler error
System.Net.WebSockets.WebSocketException (0x80004005): The WebSocket is in an invalid state ('Closed') for this operation. Valid states are: 'Open, CloseReceived'
   at System.Net.WebSockets.WebSocketValidate.ThrowIfInvalidState(WebSocketState currentState, Boolean isDisposed, WebSocketState[] validStates)
   at System.Net.WebSockets.ManagedWebSocket.SendAsync(ReadOnlyMemory`1 buffer, WebSocketMessageType messageType, WebSocketMessageFlags messageFlags, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ProcessInternal(PipeReader reader)
   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ReceiveAsync(CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)
```
These are the only error messages i see. These error came up while playing music through Feishin but playing through official client behaves same way.

**Comment by VigibotDev** [comment]
Same probleme, freeze every few hours, without Docker, on bare Debian I need to systemctl restart jellyfin :'(

**Comment by felix920506** [comment]
Is anyone here using LXC

**Comment by JonHodgins** [comment]
Experiencing the same problem on 10.9.1 using docker and the official image.

**Comment by VigibotDev** [comment]
This is a general issue that seems to make this 1.9 unusable, I'm going to wait a bit while trying to find a way to help debug. Otherwise it will be a rollback (difficult because you have to restore the database from a version 1.8 and redo a lot of work)

**Comment by JamsRepos** [comment]
> This is a general issue that seems to make this 1.9 unusable, I'm going to wait a bit while trying to find a way to help debug. Otherwise it will be a rollback (difficult because you have to restore the database from a version 1.8 and redo a lot of work)

Yeah - I rolled back to 10.8.13 and it's all perfect and responsive again. Think i'll hold off on 10.9 until it's a little bit more stable.

**Comment by soakes** [comment]
> Is anyone here using LXC If so #11344

@felix920506 I am also running LXC (Proxmox), a Debian 12 VM which runs the official jellyfin deb repo which has shared mounts from the core with some additional nfs mounts connected from inside the VM.

The only thing I have noticed which is worth mentioning is the logs shows:

```
[2024-05-14 18:22:43.368 +00:00] [WRN] The WebRootPath was not found: "/var/lib/jellyfin/wwwroot". Static files may be unavailable.
```

Confirming on the filesystem this path is incorrect and should be set to:

```
/usr/lib/jellyfin/bin/wwwroot
```

Im not seeing anything worth noting thats in the logs (yet).  Will update if I find anything else.

**Comment by Jomack16** [comment]
I am experiencing the same "locking up"
It is evidenced by streams failing, and the log ceasing to write.

The last line of the log is how I can know exactly when the problem happens, because nothing else is logged until the container is restarted.

This is the last ERR in the log before Jellyfin locks up:
[2024-05-14 13:15:46.337 -05:00] [ERR] Error processing request. URL "GET" "/Items/c1bb9e36565a7344da5f7dd6f417f08c/Images/Primary".
System.InvalidOperationException: Unable to convert any images to local

And this is the last line in the log before Jellyfin locks up:
[2024-05-14 13:15:52.511 -05:00] [DBG] "GetItemList" query time (slow): 371.5236ms. Query: "select type,data,EndDate,ChannelId,CommunityRating,CustomRating,IndexNumber,IsLocked,PreferredMetadataLanguage,PreferredMetadataCountryCode,Width,Height,DateLastRefreshed,Name,Path,PremiereDate,Overview,ParentIndexNumber,ProductionYear,OfficialRating,ForcedSortName,RunTimeTicks,Size,DateCreated,DateModified,guid,Genres,ParentId,Audio,IsInMixedFolder,DateLastSaved,LockedFields,Studios,Tags,OriginalTitle,PrimaryVersionId,DateLastMediaAdded,Album,LUFS,NormalizationGain,CriticRating,IsVirtualItem,SeriesName,PresentationUniqueKey,InheritedParentalRatingValue,ExternalSeriesId,Tagline,ProviderIds,Images,ProductionLocations,ExtraIds,TotalBitrate,ExtraType,ExternalId,SeriesPresentationUniqueKey,OwnerId from TypedBaseItems A where type=@type AND Guid in (select itemId from AncestorIds where AncestorId=@AncestorId)"

If it is useful, I can provide copies of these debug level logs.

**Comment by joshuaboniface** [comment]
I am also experiencing this occasionally, and am set up to do a full core dump of the process the next time it does. Though in my case on Debian 11 (standard `.deb` packages via repo) it's only happening once every couple days at most.

**Comment by VigibotDev** [comment]
3 time in about 1 hour

**Comment by memehammad** [comment]
> I am also experiencing this occasionally, and am set up to do a full core dump of the process the next time it does. Though in my case on Debian 11 (standard `.deb` packages via repo) it's only happening once every couple days at most.

Try having a lot of concurrent streams. I usually just opened 10 new tabs, played the same movie on each one of them. After about 5 minutes jellyfin becomes unresponsive.

**Comment by Jomack16** [comment]
> I am also experiencing this occasionally, and am set up to do a full core dump of the process the next time it does. Though in my case on Debian 11 (standard `.deb` packages via repo) it's only happening once every couple days at most.

This does only seem to occur after certain threshold of concurrent streams. I haven't nailed down the number yet, but seems to be 9+ or more.

**Comment by VigibotDev** [comment]
Do you have also a lot of :
(or it's only me ?)

```
FROM \"Users\" AS \"u\"
LEFT JOIN \"ImageInfos\" AS \"i\" ON \"u\".\"Id\" = \"i\".\"UserId\"
WHERE \"u\".\"Id\" = @__id_0
ORDER BY \"u\".\"Id\", \"i\".\"Id\"
LIMIT 1"
[2024-05-14 21:46:23.230 +02:00] [ERR] An exception occurred while iterating over the results of a query for context type '"Jellyfin.Server.Implementations.JellyfinDbContext"'."
""Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 6: 'database table is locked: ImageInfos'.
   at Microsoft.Data.Sqlite.SqliteException.ThrowExceptionForRC(Int32 rc, sqlite3 db)
   at Microsoft.Data.Sqlite.SqliteDataReader.NextResult()
   at Microsoft.Data.Sqlite.SqliteCommand.ExecuteReader(CommandBehavior behavior)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.InitializeReader(Enumerator enumerator)
   at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.MoveNext()"
Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 6: 'database table is locked: ImageInfos'.
   at Microsoft.Data.Sqlite.SqliteException.ThrowExceptionForRC(Int32 rc, sqlite3 db)
   at Microsoft.Data.Sqlite.SqliteDataReader.NextResult()
   at Microsoft.Data.Sqlite.SqliteCommand.ExecuteReader(CommandBehavior behavior)
   at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
   at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.InitializeReader(Enumerator enumerator)
   at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.MoveNext()
[2024-05-14 21:46:23.230 +02:00] [ERR] Failed executing DbCommand ("30,184"ms) [Parameters=["@__id_0='?' (DbType = Guid)"], CommandType='Text', CommandTimeout='30']"
""SELECT \"u\".\"Id\", \"u\".\"AudioLanguagePreference\", \"u\".\"AuthenticationProviderId\", \"u\".\"CastReceiverId\", \"u\".\"DisplayCollectionsView\", \"u\".\"DisplayMissingEpisodes\", \"u\".\"EnableAutoLogin\", \"u\".\"EnableLocalPassword\", \"u\".\"EnableNextEpisodeAutoPlay\", \"u\".\"EnableUserPreferenceAccess\", \"u\".\"HidePlayedInLatest\", \"u\".\"InternalId\", \"u\".\"InvalidLoginAttemptCount\", \"u\".\"LastActivityDate\", \"u\".\"LastLoginDate\", \"u\".\"LoginAttemptsBeforeLockout\", \"u\".\"MaxActiveSessions\", \"u\".\"MaxParentalAgeRating\", \"u\".\"MustUpdatePassword\", \"u\".\"Password\", \"u\".\"PasswordResetProviderId\", \"u\".\"PlayDefaultAudioTrack\", \"u\".\"RememberAudioSelections\", \"u\".\"RememberSubtitleSelections\", \"u\".\"RemoteClientBitrateLimit\", \"u\".\"RowVersion\", \"u\".\"SubtitleLanguagePreference\", \"u\".\"SubtitleMode\", \"u\".\"SyncPlayAccess\", \"u\".\"Username\", \"i\".\"Id\", \"i\".\"LastModified\", \"i\".\"Path\", \"i\".\"UserId\"
FROM \"Users\" AS \"u\"
```

**Comment by soakes** [comment]
> Do you have also a lot of : (or it's only me ?)
> 
> ```
> FROM \"Users\" AS \"u\"
> LEFT JOIN \"ImageInfos\" AS \"i\" ON \"u\".\"Id\" = \"i\".\"UserId\"
> WHERE \"u\".\"Id\" = @__id_0
> ORDER BY \"u\".\"Id\", \"i\".\"Id\"
> LIMIT 1"
> [2024-05-14 21:46:23.230 +02:00] [ERR] An exception occurred while iterating over the results of a query for context type '"Jellyfin.Server.Implementations.JellyfinDbContext"'."
> ""Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 6: 'database table is locked: ImageInfos'.
>    at Microsoft.Data.Sqlite.SqliteException.ThrowExceptionForRC(Int32 rc, sqlite3 db)
>    at Microsoft.Data.Sqlite.SqliteDataReader.NextResult()
>    at Microsoft.Data.Sqlite.SqliteCommand.ExecuteReader(CommandBehavior behavior)
>    at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
>    at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.InitializeReader(Enumerator enumerator)
>    at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.MoveNext()"
> Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 6: 'database table is locked: ImageInfos'.
>    at Microsoft.Data.Sqlite.SqliteException.ThrowExceptionForRC(Int32 rc, sqlite3 db)
>    at Microsoft.Data.Sqlite.SqliteDataReader.NextResult()
>    at Microsoft.Data.Sqlite.SqliteCommand.ExecuteReader(CommandBehavior behavior)
>    at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
>    at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.InitializeReader(Enumerator enumerator)
>    at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.MoveNext()
> [2024-05-14 21:46:23.230 +02:00] [ERR] Failed executing DbCommand ("30,184"ms) [Parameters=["@__id_0='?' (DbType = Guid)"], CommandType='Text', CommandTimeout='30']"
> ""SELECT \"u\".\"Id\", \"u\".\"AudioLanguagePreference\", \"u\".\"AuthenticationProviderId\", \"u\".\"CastReceiverId\", \"u\".\"DisplayCollectionsView\", \"u\".\"DisplayMissingEpisodes\", \"u\".\"EnableAutoLogin\", \"u\".\"EnableLocalPassword\", \"u\".\"EnableNextEpisodeAutoPlay\", \"u\".\"EnableUserPreferenceAccess\", \"u\".\"HidePlayedInLatest\", \"u\".\"InternalId\", \"u\".\"InvalidLoginAttemptCount\", \"u\".\"LastActivityDate\", \"u\".\"LastLoginDate\", \"u\".\"LoginAttemptsBeforeLockout\", \"u\".\"MaxActiveSessions\", \"u\".\"MaxParentalAgeRating\", \"u\".\"MustUpdatePassword\", \"u\".\"Password\", \"u\".\"PasswordResetProviderId\", \"u\".\"PlayDefaultAudioTrack\", \"u\".\"RememberAudioSelections\", \"u\".\"RememberSubtitleSelections\", \"u\".\"RemoteClientBitrateLimit\", \"u\".\"RowVersion\", \"u\".\"SubtitleLanguagePreference\", \"u\".\"SubtitleMode\", \"u\".\"SyncPlayAccess\", \"u\".\"Username\", \"i\".\"Id\", \"i\".\"LastModified\", \"i\".\"Path\", \"i\".\"UserId\"
> FROM \"Users\" AS \"u\"
> ```

I haven't got anything like this and yes it seems to be dying several times an hour.

**Comment by memehammad** [comment]
> Do you have also a lot of : (or it's only me ?)
> 
> ```
> FROM \"Users\" AS \"u\"
> LEFT JOIN \"ImageInfos\" AS \"i\" ON \"u\".\"Id\" = \"i\".\"UserId\"
> WHERE \"u\".\"Id\" = @__id_0
> ORDER BY \"u\".\"Id\", \"i\".\"Id\"
> LIMIT 1"
> [2024-05-14 21:46:23.230 +02:00] [ERR] An exception occurred while iterating over the results of a query for context type '"Jellyfin.Server.Implementations.JellyfinDbContext"'."
> ""Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 6: 'database table is locked: ImageInfos'.
>    at Microsoft.Data.Sqlite.SqliteException.ThrowExceptionForRC(Int32 rc, sqlite3 db)
>    at Microsoft.Data.Sqlite.SqliteDataReader.NextResult()
>    at Microsoft.Data.Sqlite.SqliteCommand.ExecuteReader(CommandBehavior behavior)
>    at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
>    at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.InitializeReader(Enumerator enumerator)
>    at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.MoveNext()"
> Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 6: 'database table is locked: ImageInfos'.
>    at Microsoft.Data.Sqlite.SqliteException.ThrowExceptionForRC(Int32 rc, sqlite3 db)
>    at Microsoft.Data.Sqlite.SqliteDataReader.NextResult()
>    at Microsoft.Data.Sqlite.SqliteCommand.ExecuteReader(CommandBehavior behavior)
>    at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
>    at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.InitializeReader(Enumerator enumerator)
>    at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.MoveNext()
> [2024-05-14 21:46:23.230 +02:00] [ERR] Failed executing DbCommand ("30,184"ms) [Parameters=["@__id_0='?' (DbType = Guid)"], CommandType='Text', CommandTimeout='30']"
> ""SELECT \"u\".\"Id\", \"u\".\"AudioLanguagePreference\", \"u\".\"AuthenticationProviderId\", \"u\".\"CastReceiverId\", \"u\".\"DisplayCollectionsView\", \"u\".\"DisplayMissingEpisodes\", \"u\".\"EnableAutoLogin\", \"u\".\"EnableLocalPassword\", \"u\".\"EnableNextEpisodeAutoPlay\", \"u\".\"EnableUserPreferenceAccess\", \"u\".\"HidePlayedInLatest\", \"u\".\"InternalId\", \"u\".\"InvalidLoginAttemptCount\", \"u\".\"LastActivityDate\", \"u\".\"LastLoginDate\", \"u\".\"LoginAttemptsBeforeLockout\", \"u\".\"MaxActiveSessions\", \"u\".\"MaxParentalAgeRating\", \"u\".\"MustUpdatePassword\", \"u\".\"Password\", \"u\".\"PasswordResetProviderId\", \"u\".\"PlayDefaultAudioTrack\", \"u\".\"RememberAudioSelections\", \"u\".\"RememberSubtitleSelections\", \"u\".\"RemoteClientBitrateLimit\", \"u\".\"RowVersion\", \"u\".\"SubtitleLanguagePreference\", \"u\".\"SubtitleMode\", \"u\".\"SyncPlayAccess\", \"u\".\"Username\", \"i\".\"Id\", \"i\".\"LastModified\", \"i\".\"Path\", \"i\".\"UserId\"
> FROM \"Users\" AS \"u\"
> ```

Yes, I've gotten this a few times. Not always, but often.

**Comment by VigibotDev** [comment]
It's look like I get this on every crash time, but I'm not sure

**Comment by Jomack16** [comment]
> Do you have also a lot of : (or it's only me ?)
> 
> ```
> FROM \"Users\" AS \"u\"
> LEFT JOIN \"ImageInfos\" AS \"i\" ON \"u\".\"Id\" = \"i\".\"UserId\"
> WHERE \"u\".\"Id\" = @__id_0
> ORDER BY \"u\".\"Id\", \"i\".\"Id\"
> LIMIT 1"
> [2024-05-14 21:46:23.230 +02:00] [ERR] An exception occurred while iterating over the results of a query for context type '"Jellyfin.Server.Implementations.JellyfinDbContext"'."
> ""Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 6: 'database table is locked: ImageInfos'.
>    at Microsoft.Data.Sqlite.SqliteException.ThrowExceptionForRC(Int32 rc, sqlite3 db)
>    at Microsoft.Data.Sqlite.SqliteDataReader.NextResult()
>    at Microsoft.Data.Sqlite.SqliteCommand.ExecuteReader(CommandBehavior behavior)
>    at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
>    at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.InitializeReader(Enumerator enumerator)
>    at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.MoveNext()"
> Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 6: 'database table is locked: ImageInfos'.
>    at Microsoft.Data.Sqlite.SqliteException.ThrowExceptionForRC(Int32 rc, sqlite3 db)
>    at Microsoft.Data.Sqlite.SqliteDataReader.NextResult()
>    at Microsoft.Data.Sqlite.SqliteCommand.ExecuteReader(CommandBehavior behavior)
>    at Microsoft.EntityFrameworkCore.Storage.RelationalCommand.ExecuteReader(RelationalCommandParameterObject parameterObject)
>    at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.InitializeReader(Enumerator enumerator)
>    at Microsoft.EntityFrameworkCore.Query.Internal.SplitQueryingEnumerable`1.Enumerator.MoveNext()
> [2024-05-14 21:46:23.230 +02:00] [ERR] Failed executing DbCommand ("30,184"ms) [Parameters=["@__id_0='?' (DbType = Guid)"], CommandType='Text', CommandTimeout='30']"
> ""SELECT \"u\".\"Id\", \"u\".\"AudioLanguagePreference\", \"u\".\"AuthenticationProviderId\", \"u\".\"CastReceiverId\", \"u\".\"DisplayCollectionsView\", \"u\".\"DisplayMissingEpisodes\", \"u\".\"EnableAutoLogin\", \"u\".\"EnableLocalPassword\", \"u\".\"EnableNextEpisodeAutoPlay\", \"u\".\"EnableUserPreferenceAccess\", \"u\".\"HidePlayedInLatest\", \"u\".\"InternalId\", \"u\".\"InvalidLoginAttemptCount\", \"u\".\"LastActivityDate\", \"u\".\"LastLoginDate\", \"u\".\"LoginAttemptsBeforeLockout\", \"u\".\"MaxActiveSessions\", \"u\".\"MaxParentalAgeRating\", \"u\".\"MustUpdatePassword\", \"u\".\"Password\", \"u\".\"PasswordResetProviderId\", \"u\".\"PlayDefaultAudioTrack\", \"u\".\"RememberAudioSelections\", \"u\".\"RememberSubtitleSelections\", \"u\".\"RemoteClientBitrateLimit\", \"u\".\"RowVersion\", \"u\".\"SubtitleLanguagePreference\", \"u\".\"SubtitleMode\", \"u\".\"SyncPlayAccess\", \"u\".\"Username\", \"i\".\"Id\", \"i\".\"LastModified\", \"i\".\"Path\", \"i\".\"UserId\"
> FROM \"Users\" AS \"u\"
> ```

I've only seen this once in the Debug level logging I have been collecting.
During the same time-period I have had to restart many times.
So they don't seem to correlate for me.

**Comment by joshuaboniface** [comment]
I don't get those DB locks either, so it might be a separate failure. There is another issue for those.

**Comment by Jomack16** [comment]
> > I am also experiencing this occasionally, and am set up to do a full core dump of the process the next time it does. Though in my case on Debian 11 (standard `.deb` packages via repo) it's only happening once every couple days at most.
> 
> This does only seem to occur after certain threshold of concurrent streams. I haven't nailed down the number yet, but seems to be 9+ or more.

After some testing, It looks like the number is more like 25 concurrent, before another tab would not load.
However I did not get it to stop logging, so if the cause is the number of concurrent streams, It may take more than that.

**Comment by memehammad** [comment]
> After some testing, It looks like the number is more like 25 concurrent, before another tab would not load. However I did not get it to stop logging, so if the cause is the number of concurrent streams, It may take more than that.

Pretty sure it depends on how powerful your system is. I'm on a low power system (Intel j4125) and I've gotten these issues with as few as 3 streams.

**Comment by soakes** [comment]
> > After some testing, It looks like the number is more like 25 concurrent, before another tab would not load. However I did not get it to stop logging, so if the cause is the number of concurrent streams, It may take more than that.
> 
> Pretty sure it depends on how powerful your system is. I'm on a low power system (Intel j4125) and I've gotten these issues with as few as 3 streams.

Getting the issue on a **12th Gen Intel(R) Core(TM) i5-1235U** with iGPU, even with just a single stream, so I dont think stream count/specs are really the cause.

**Comment by VigibotDev** [comment]
And another lock, more rare (without ImageInfos) I try to correlate if it's near my systemctl restarts:
```
""Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 6: 'database table is locked'.
   at Microsoft.Data.Sqlite.SqliteDataReader.NextResult()
   at Microsoft.Data.Sqlite.SqliteCommand.ExecuteReader(CommandBehavior behavior)
   at Microsoft.Data.Sqlite.SqliteConnectionExtensions.ExecuteNonQuery(SqliteConnection connection, String commandText, SqliteParameter[] parameters)
   at Microsoft.Data.Sqlite.SqliteTransaction..ctor(SqliteConnection connection, IsolationLevel isolationLevel, Boolean deferred)
   at System.Data.Common.DbConnection.BeginDbTransactionAsync(IsolationLevel isolationLevel, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.BeginTransactionAsync(IsolationLevel isolationLevel, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Storage.RelationalConnection.BeginTransactionAsync(CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.Update.Internal.BatchExecutor.ExecuteAsync(IEnumerable`1 commandBatches, IRelationalConnection connection, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(IList`1 entriesToSave, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChangesAsync(StateManager stateManager, Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)"
```

**Comment by VigibotDev** [comment]
certainly a consequence but not the cause:(

**Comment by joshuaboniface** [comment]
The times I've noticed this, I always only had a very small number of concurrent streams (1, maybe 2) and testing with 10 now so far no crash/lockup. I'll let those run for a little bit longer.

**Comment by VigibotDev** [comment]
Oh, crashed exactly when I close my two fake load player !

(client log) 
```
Unexpected Application Error!
Loading chunk 80187 failed. (error: https://www.serveurperso.com/jellyfin/web/search.6fcb4b64dc5ad5262510.chunk.js)

s.f.j@https://www.serveurperso.com/jellyfin/web/runtime.bundle.js?a32cc7cf54b43478bbe4:1:46403
s.e/<@https://www.serveurperso.com/jellyfin/web/runtime.bundle.js?a32cc7cf54b43478bbe4:1:1410
s.e@https://www.serveurperso.com/jellyfin/web/runtime.bundle.js?a32cc7cf54b43478bbe4:1:1375
a@https://www.serveurperso.com/jellyfin/web/94047.515d4f5ba699aeed4cac.chunk.js:1:59854
65939/l<@https://www.serveurperso.com/jellyfin/web/94047.515d4f5ba699aeed4cac.chunk.js:1:36582
_@https://www.serveurperso.com/jellyfin/web/99377.bundle.js?a32cc7cf54b43478bbe4:2:53274
98165/h/b/n.resolveAsync@https://www.serveurperso.com/jellyfin/web/99377.bundle.js?a32cc7cf54b43478bbe4:2:55607
98165/h/b/n.loadAsync@https://www.serveurperso.com/jellyfin/web/99377.bundle.js?a32cc7cf54b43478bbe4:2:55260
98165/h/b/n.componentDidMount@https://www.serveurperso.com/jellyfin/web/99377.bundle.js?a32cc7cf54b43478bbe4:2:54377
mu@https://www.serveurperso.com/jellyfin/web/node_modules.react-dom.bundle.js?a32cc7cf54b43478bbe4:2:85704
Ti@https://www.serveurperso.com/jellyfin/web/node_modules.react-dom.bundle.js?a32cc7cf54b43478bbe4:2:102401
77699/t.unstable_runWithPriority@https://www.serveurperso.com/jellyfin/web/99377.bundle.js?a32cc7cf54b43478bbe4:2:41417
jl@https://www.serveurperso.com/jellyfin/web/node_modules.react-dom.bundle.js?a32cc7cf54b43478bbe4:2:44769
Li@https://www.serveurperso.com/jellyfin/web/node_modules.react-dom.bundle.js?a32cc7cf54b43478bbe4:2:99149
vi@https://www.serveurperso.com/jellyfin/web/node_modules.react-dom.bundle.js?a32cc7cf54b43478bbe4:2:95282
76067/ql/<@https://www.serveurperso.com/jellyfin/web/node_modules.react-dom.bundle.js?a32cc7cf54b43478bbe4:2:44992
77699/t.unstable_runWithPriority@https://www.serveurperso.com/jellyfin/web/99377.bundle.js?a32cc7cf54b43478bbe4:2:41417
jl@https://www.serveurperso.com/jellyfin/web/node_modules.react-dom.bundle.js?a32cc7cf54b43478bbe4:2:44769
ql@https://www.serveurperso.com/jellyfin/web/node_modules.react-dom.bundle.js?a32cc7cf54b43478bbe4:2:44939
Kl@https://www.serveurperso.com/jellyfin/web/node_modules.react-dom.bundle.js?a32cc7cf54b43478bbe4:2:44872
di@https://www.serveurperso.com/jellyfin/web/node_modules.react-dom.bundle.js?a32cc7cf54b43478bbe4:2:92476
Lo@https://www.serveurperso.com/jellyfin/web/node_modules.react-dom.bundle.js?a32cc7cf54b43478bbe4:2:64362
9055/O/G</<@https://www.serveurperso.com/jellyfin/web/node_modules.react-router-dom.1c11d916c27725770829.chunk.js:2:12812
9055/O/U<@https://www.serveurperso.com/jellyfin/web/node_modules.react-router-dom.1c11d916c27725770829.chunk.js:2:11800
9055/O/G<@https://www.serveurperso.com/jellyfin/web/node_modules.react-router-dom.1c11d916c27725770829.chunk.js:2:12791
87383/ke/he/<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:25837
he@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:25809
de@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:27126
87383/ke/_e/t/<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:32389
d@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:730
87383/a/P/<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:2073
87383/a/L/</<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:1159
o@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:6633
u@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:6837
87383/i/</<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:6896
87383/i/<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:6776
Ae@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:31855
87383/ke/Re/t/<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:30227
d@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:730
87383/a/P/<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:2073
87383/a/L/</<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:1159
o@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:6633
u@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:6837
87383/i/</<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:6896
87383/i/<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:6776
Le@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:28840
87383/ke/ke/t/<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:28559
d@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:730
87383/a/P/<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:2073
87383/a/L/</<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:1159
o@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:6633
u@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:6837
87383/i/</<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:6896
87383/i/<@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:6776
pe@https://www.serveurperso.com/jellyfin/web/node_modules.@remix-run.router.33c9ecbab57840d1d963.chunk.js:2:27427
63940/l/</e</</<@https://www.serveurperso.com/jellyfin/web/94047.515d4f5ba699aeed4cac.chunk.js:1:49563
63940/o/s/</<@https://www.serveurperso.com/jellyfin/web/94047.515d4f5ba699aeed4cac.chunk.js:1:48340
63940/o/s/<@https://www.serveurperso.com/jellyfin/web/94047.515d4f5ba699aeed4cac.chunk.js:1:48445
63940/i/<@https://www.serveurperso.com/jellyfin/web/94047.515d4f5ba699aeed4cac.chunk.js:1:47360
i@https://www.serveurperso.com/jellyfin/web/94047.515d4f5ba699aeed4cac.chunk.js:1:47105
63940/l/</e<@https://www.serveurperso.com/jellyfin/web/94047.515d4f5ba699aeed4cac.chunk.js:1:48909
25127/l/call/<@https://www.serveurperso.com/jellyfin/web/node_modules.history.bundle.js?a32cc7cf54b43478bbe4:1:2479
call@https://www.serveurperso.com/jellyfin/web/node_modules.history.bundle.js?a32cc7cf54b43478bbe4:1:2448
L@https://www.serveurperso.com/jellyfin/web/node_modules.history.bundle.js?a32cc7cf54b43478bbe4:1:1525
n@https://www.serveurperso.com/jellyfin/web/node_modules.history.bundle.js?a32cc7cf54b43478bbe4:1:1865
70267/e/</this.promiseShow</<@https://www.serveurperso.com/jellyfin/web/main.jellyfin.bundle.js?a32cc7cf54b43478bbe4:2:197037
```

**Comment by VigibotDev** [comment]
To recover some of my crash timestamp tonight (because it can crash with less than 9 WS) :

(root|/var/log/jellyfin) grep "Lost .. WebSockets." jellyfin20240514.log
[2024-05-14 19:56:33.725 +02:00] [INF] Lost 48 WebSockets.
[2024-05-14 21:27:56.001 +02:00] [INF] Lost 45 WebSockets.
[2024-05-14 21:43:38.231 +02:00] [INF] Lost 42 WebSockets.
grep "Lost .. WebSockets." jellyfin20240514_001.log
[2024-05-14 22:52:45.837 +02:00] [INF] Lost 50 WebSockets.

**Comment by VigibotDev** [comment]
inside /etc/jellyfin/logging.json change "MinimumLevel": "Information" to :
{
    "Serilog": {
        "MinimumLevel": "Debug",
        "WriteTo": [
            {
...

**Comment by SingingFrog7** [comment]
> WAITING for a crash !

I have a similar issue that was closed as duplicate of #11344 
I had nothing useful in the log at debug level either, some people said that the issue started between version 20240104.7 and 20240112.1 so it has to be a commit between those dates? A lot was done between those dates, but Dotnet was updated to 8.0.1, I can't say if it could be the cause. If I can spare some time later, I'll try to build the 10.9.1 with Dotnet 8.0.0 and see what it gives

**Comment by soakes** [comment]
> inside /etc/jellyfin/logging.json change "MinimumLevel": "Information" to : { "Serilog": { "MinimumLevel": "Debug", "WriteTo": [ { ... WAITING for a crash !

Also tried, lasted about 20 minutes then died, nothing in logs which is worth noting.

What is intresting is, jellyfin itself is still alive, log was still processing info, port was still open but if you try and curl to it, it just hangs.

**Comment by VigibotDev** [comment]
I deleted all sessions because some users can't connect... resulting in a total crash of all client (no login redirect).... can't debug anymore

**Comment by derfy79** [comment]
Same probleme, freeze every few hours, official linux docker on unraid

**Comment by joshuaboniface** [comment]
OK I was able to reproduce this myself finally and get some core dumps for analysis.

Curiously this time though, it didn't *completely* lock up. It was still kinda responding, with lots of

```
May 14 23:24:07 jf1.p.bonilan.net jellyfin[616]: [ERR] Error reporting playback progress
May 14 23:24:07 jf1.p.bonilan.net jellyfin[616]: MediaBrowser.Common.Extensions.ResourceNotFoundException: Session a5385ff9d3bf89070e9da99d27f46da7 not found.
May 14 23:24:07 jf1.p.bonilan.net jellyfin[616]:    at Emby.Server.Implementations.Session.SessionManager.OnPlaybackProgress(PlaybackProgressInfo info, Boolean isAutomated)
May 14 23:24:07 jf1.p.bonilan.net jellyfin[616]:    at MediaBrowser.Controller.Session.SessionInfo.OnProgressTimerCallback(Object state)
May 14 23:24:07 jf1.p.bonilan.net jellyfin[616]: [ERR] Error reporting playback progress
May 14 23:24:07 jf1.p.bonilan.net jellyfin[616]: MediaBrowser.Common.Extensions.ResourceNotFoundException: Session a5385ff9d3bf89070e9da99d27f46da7 not found.
May 14 23:24:07 jf1.p.bonilan.net jellyfin[616]:    at Emby.Server.Implementations.Session.SessionManager.OnPlaybackProgress(PlaybackProgressInfo info, Boolean isAutomated)
May 14 23:24:07 jf1.p.bonilan.net jellyfin[616]:    at MediaBrowser.Controller.Session.SessionInfo.OnProgressTimerCallback(Object state)
```

And then eventually after about ~5 minutes it started responding again.

**Comment by joshuaboniface** [comment]
For anyone this is happening to, does Jellyfin *eventually* start responding again? Presumably like me you restart fairly quickly, but I'm curious if it's left, if it starts responding on its own again after any reasonable (~5, 10, 15 mins) time.

**Comment by SingingFrog7** [comment]
At least once for me, it did start responding again after some time

Weirdly, I had no crash/lock during the last 24h, but had about 10 occurence yesterday

**Comment by gyilkoszabpehely** [comment]
Hi guys! I also have the same problem, yesterday I found something, try these steps maybe will help to find the problem:
Start monitoring your server, Uptime Kuma is good option for that, thats the url you need to watch http://YOURIP:8096/health

I found nothing relevant in the logs so I removed all scheduled tasks, it hasn't frozen since...
Maybe it's a library scan problem but I doesn't understand why can't see anything in logs because of that.

**Comment by ShakeSp33r** [comment]
Disabled all schedules tasks, and system up for the last 12 hours. 
Had 4 streams running and all good, previously on 10.8.13 I did 7/8 streams (Direct Play & transcoding) with no issues.

Will see how it goes

**Comment by haath** [comment]
+1

I get random freezes both while streaming videos and while only streaming music.
With a cursory look through the docker logs I can only spot "websocket lost" messages that others reported.

Sometimes it lasts a couple of minutes, other times I have to restart the container.

Ubuntu 23.10 x86_64 on Intel Celeron N3350

**Comment by VigibotDev** [comment]
Rollback for me.
It's not in Debian to push a beta software inside a stable repository. Server crash every time, library scan fail without reason, every users must uninstall/reinstall app each time they want connect and it's work only once...

**Comment by JamsRepos** [comment]
> Rollback for me. It's not in Debian to push a beta software inside a stable repository. Server crash every time, library scan fail without reason, every users must uninstall/reinstall app each time they want connect and it's work only once...

I get your frustration, mine too. I also did a rollback however you've got to realise that you cannot do mass user testing without pushing things up the pipeline. You can only find so much in a restricted and limited test environment. 

These things happen, if you want stability then don't upgrade Day 1.

**Comment by soakes** [comment]
Thanks for everyone whos added comments to this issue, it helps to try and narrow down the real issue.  This issue is hard to find due to logs are not being very helpful.

I have however, been doing some testing myself, what ive found is the following:

- If you have no users connect, over a period of about 8 or so hours (sleeping), system is stable (still up when get up)
- I don't believe its a scheduled job causing this, as initially when the problem showed, it was one of the first things I checked (manually ran all jobs, even tried to run all at same time to see if triggers a crash).
- What I have found is, if you just have a web client (i.e. firefox) watching a show (~42 mins), it stays stable.  I suggest others try and make sure no android/iOS clients connect and then try just on a browser and see if its stable or not.  This may lead closer to the real probelm. 
- If you watch on an android client (i.e Pixel 6 Pro) as well as watch with Firefox (mac laptop), both clients will crash/hang about ~9-15 mins later.
- Waiting for ~15 mins to recover doesn't work (now 20 mins as of this report).  

Will do some further testing i.e single client to make sure this was not a fluke and will report/update this later.

**Comment by VigibotDev** [comment]
Can you try to delete a client session, and reconnect ? this require an app reinstall !

**Comment by soakes** [comment]
> Can you try to delete a client session, and reconnect ? this require an app reinstall !

That would be a little problematic here as theres several external "non tech" users, but I might be able to sort something out.

**Comment by gyilkoszabpehely** [comment]
I turned on my backup server and created a clean 10.9.1 install, new library scan to rule out that the existing database is not the cause of the problem. No problem until the fifth test user start playing, all of these from web client.
No response from jellyfin, server ok no load , log stopped and after 20 min. new lines came up in log file but still nothing relevant.
`[2024-05-15 10:40:52.024 +02:00] [INF] Playback stopped reported by app "Jellyfin Web" "10.9.1" playing "". Stopped at "326836" ms
[2024-05-15 10:55:52.454 +02:00] [INF] Playback stopped reported by app "Jellyfin Web" "10.9.1" playing "". Stopped at "631894" ms
[2024-05-15 11:00:51.857 +02:00] [INF] Playback stopped reported by app "Jellyfin Web" "10.9.1" playing "". Stopped at "454263" ms`
From the timestamps you can see there is a big delay between new lines, I still waiting what will happen after more time...

**Comment by soakes** [comment]
> I turned on my backup server and created a clean 10.9.1 install, new library scan to rule out that the existing database is not the cause of the problem. No problem until the fifth test user start playing, all of these from web client. No response from jellyfin, server ok no load , log stopped and after 20 min. new lines came up in log file but still nothing relevant. `[2024-05-15 10:40:52.024 +02:00] [INF] Playback stopped reported by app "Jellyfin Web" "10.9.1" playing "". Stopped at "326836" ms [2024-05-15 10:55:52.454 +02:00] [INF] Playback stopped reported by app "Jellyfin Web" "10.9.1" playing "". Stopped at "631894" ms [2024-05-15 11:00:51.857 +02:00] [INF] Playback stopped reported by app "Jellyfin Web" "10.9.1" playing "". Stopped at "454263" ms` From the timestamps you can see there is a big delay between new lines, I still waiting what will happen after more time...

One question @gyilkoszabpehely, is this using the same database/config? or was this a completly fresh install i.e no data copied over (apart from accessing movies/tv)

Initially when reading your comment, I thought it was a full reinstall, but you also mentioned "new lib scan to rule out existing db" and so I now suspect you just have uninstalled jellyfin and installed 10.9.1 again.

Only asking as one of the things i've yet to test is a fresh VM, with fresh config/db and only old data would be the tv/movie data itself.  This would be a good test to iron out if its an upgrade "thing" thats broken or it is jellyfin itself.

On a completly seperate note, since my last update, second attempt with Firefox and an AndroidTV, hung jellyfin about ~11 minutes later.  

I have currently a Firefox and Chrome user online (nothing else) and its still stable (time roughly just hitting 17 minutes), will see if this completes.

**Comment by gyilkoszabpehely** [comment]
> > I turned on my backup server and created a clean 10.9.1 install, new library scan to rule out that the existing database is not the cause of the problem. No problem until the fifth test user start playing, all of these from web client. No response from jellyfin, server ok no load , log stopped and after 20 min. new lines came up in log file but still nothing relevant. `[2024-05-15 10:40:52.024 +02:00] [INF] Playback stopped reported by app "Jellyfin Web" "10.9.1" playing "". Stopped at "326836" ms [2024-05-15 10:55:52.454 +02:00] [INF] Playback stopped reported by app "Jellyfin Web" "10.9.1" playing "". Stopped at "631894" ms [2024-05-15 11:00:51.857 +02:00] [INF] Playback stopped reported by app "Jellyfin Web" "10.9.1" playing "". Stopped at "454263" ms` From the timestamps you can see there is a big delay between new lines, I still waiting what will happen after more time...
> 
> One question @gyilkoszabpehely, is this using the same database/config? or was this a completly fresh install i.e no data copied over (apart from accessing movies/tv)
> 
> Initially when reading your comment, I thought it was a full reinstall, but you also mentioned "new lib scan to rule out existing db" and so I now suspect you just have uninstalled jellyfin and installed 10.9.1 again.
> 
> Only asking as one of the things i've yet to test is a fresh VM, with fresh config/db and only old data would be the tv/movie data itself. This would be a good test to iron out if its an upgrade "thing" thats broken or it is jellyfin itself.
> 
> On a completly seperate note, since my last update, second attempt with Firefox and an AndroidTV, hung jellyfin about ~11 minutes later.
> 
> I have currently a Firefox and Chrome user online (nothing else) and its still stable (time roughly just hitting 17 minutes), will see if this completes.

My my live server is updated with an existing database.
My backup server is clean installed this morning to test out problems...

**Comment by soakes** [comment]
> My my live server is updated with an existing database. My backup server is clean installed this morning to test out problems...

Thank you, this is helpful, saves me a job :)

Update, just as you posted this message, my two clients now just crashed, roughly 20 minutes in.  These were Firefox and Chrone.  So I think that rules out android/ios issues talking to new server.

**Comment by gyilkoszabpehely** [comment]
I can confirm that delete scheduled tasks will not solve the problem. 
I also tried to switch from hardware encoding to software but problem still exist.
I collected all error from log what happend in the past minutes:
```
[WRN] "MYIP": External request received, no matching external bind address found, trying internal addresses.
[WRN] "MYIP": External request received, but no external interface found. Need to route through internal network.
[ERR] Error processing request: "Unexpected end of request content". URL "POST" "/Sessions/Playing/Progress".
[WRN] The WebRootPath was not found: "/var/lib/jellyfin/wwwroot". Static files may be unavailable.
[WRN] WS "MYIP" error receiving data: "The remote party closed the WebSocket connection without completing the close handshake."
[ERR] Error processing request: "Unexpected end of request content". URL "POST" "/Sessions/Capabilities/Full".
```

**Comment by soakes** [comment]
> I can confirm that delete scheduled tasks will not solve the problem. I also tried to switch from hardware encoding to software but problem still exist. I collected all error from log what happend in the past minutes:
> 
> ```
> [WRN] "MYIP": External request received, no matching external bind address found, trying internal addresses.
> [WRN] "MYIP": External request received, but no external interface found. Need to route through internal network.
> [ERR] Error processing request: "Unexpected end of request content". URL "POST" "/Sessions/Playing/Progress".
> [WRN] The WebRootPath was not found: "/var/lib/jellyfin/wwwroot". Static files may be unavailable.
> [WRN] WS "MYIP" error receiving data: "The remote party closed the WebSocket connection without completing the close handshake."
> [ERR] Error processing request: "Unexpected end of request content". URL "POST" "/Sessions/Capabilities/Full".
> ```

I just updated https://github.com/jellyfin/jellyfin/issues/11344#issuecomment-2112124780 issue as @Schutzwurst  suggested that thread count was going crazy and hes right.  Added some screenshots.

Whats odd is, while the count is going crazy, theres nothing in the logs.  Also whats very odd is, I just had another crash, was down for 5 or so mins and this time it recovered itself and even the thread count returned to normal (~30).

**Comment by DwayneGoddem** [comment]
I having the same sort of issue but on Windows hosting Jellyfin after upgrading to 10.9.1.
its driving me nuts.

Same nothing in the logs or anything.

**Comment by DwayneGoddem** [comment]
I just got this in the log on a crash.

[2024-05-15 21:30:50.617 +10:00] [ERR] [17] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.0.84" WebSocketRequestHandler error
System.Net.WebSockets.WebSocketException (997): The WebSocket is in an invalid state ('Closed') for this operation. Valid states are: 'Open, CloseReceived'
   at System.Net.WebSockets.ManagedWebSocket.SendAsync(ReadOnlyMemory`1 buffer, WebSocketMessageType messageType, WebSocketMessageFlags messageFlags, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ProcessInternal(PipeReader reader)
   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ReceiveAsync(CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)
[2024-05-15 21:30:50.621 +10:00] [ERR] [17] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "192.168.0.92" WebSocketRequestHandler error
System.Net.WebSockets.WebSocketException (997): The WebSocket is in an invalid state ('Closed') for this operation. Valid states are: 'Open, CloseReceived'
   at System.Net.WebSockets.ManagedWebSocket.SendAsync(ReadOnlyMemory`1 buffer, WebSocketMessageType messageType, WebSocketMessageFlags messageFlags, CancellationToken cancellationToken)
--- End of stack trace from previous location ---
   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ProcessInternal(PipeReader reader)
   at Emby.Server.Implementations.HttpServer.WebSocketConnection.ReceiveAsync(CancellationToken cancellationToken)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)
   at Emby.Server.Implementations.HttpServer.WebSocketManager.WebSocketRequestHandler(HttpContext context)
[2024-05-15 21:30:50.624 +10:00] [INF] [17] MediaBrowser.MediaEncoding.Transcoding.TranscodeManager: Transcoding kill timer stopped for JobId "cd1cec41c44e4fff873f8fe83443bbfa" PlaySessionId "027d9fc5aa7b4401ac392a7967e85da7". Killing transcoding

**Comment by gyilkoszabpehely** [comment]
Ok I reproduced this second time, someone please also test this for me?
Restart jellyfin service if hangs, copy your jellyfin url and open many tab you can do and your server can handle and play random videos from your library, I opened 14 playback and start watching in two ssh session top and btop.
After 10-15 minutes all ffmpeg process will gone, jellyfin threads drops below 20 and after that freeze coming and threads start growing.
It doesn't matter how many playback you have, the time is always the same....
I also removed all tasks from data folder and disabled RealtimeMonitor and LUFSScan in options.xml.

**Comment by gyilkoszabpehely** [comment]
> > I can confirm that delete scheduled tasks will not solve the problem. I also tried to switch from hardware encoding to software but problem still exist. I collected all error from log what happend in the past minutes:
> > ```
> > [WRN] "MYIP": External request received, no matching external bind address found, trying internal addresses.
> > [WRN] "MYIP": External request received, but no external interface found. Need to route through internal network.
> > [ERR] Error processing request: "Unexpected end of request content". URL "POST" "/Sessions/Playing/Progress".
> > [WRN] The WebRootPath was not found: "/var/lib/jellyfin/wwwroot". Static files may be unavailable.
> > [WRN] WS "MYIP" error receiving data: "The remote party closed the WebSocket connection without completing the close handshake."
> > [ERR] Error processing request: "Unexpected end of request content". URL "POST" "/Sessions/Capabilities/Full".
> > ```
> 
> I just updated [#11344 (comment)](https://github.com/jellyfin/jellyfin/issues/11344#issuecomment-2112124780) issue as @Schutzwurst suggested that thread count was going crazy and hes right. Added some screenshots.
> 
> Whats odd is, while the count is going crazy, theres nothing in the logs. Also whats very odd is, I just had another crash, was down for 5 or so mins and this time it recovered itself and even the thread count returned to normal (~30).

.NET TP Worker which starts to multiply

**Comment by joshuaboniface** [comment]
We've found quite a bit of info in our internal chats about this (going through my core dumps and logs).

The short version is, right now we're pretty sure the issue has to do with transcoding jobs and updating the transcode status. This happens very frequently, but there is a database query in there which is very heavy and after enough time, locks everything up. That TP Worker is the process running the queries.

We're working on a solution so please remain patient and we'll hopefully get that out in a 10.9.2 sometime very soon.

**Comment by gyilkoszabpehely** [comment]
> We've found quite a bit of info in our internal chats about this (going through my core dumps and logs).
> 
> The short version is, right now we're pretty sure the issue has to do with transcoding jobs and updating the transcode status. This happens very frequently, but there is a database query in there which is very heavy and after enough time the backlog locks everything up. That TP Worker is the process running the queries.
> 
> We're working on a solution so please remain patient and we'll hopefully get that out in a 10.9.2 sometime very soon.

I can't speak for others but I have to say thank you! My team really appreciate what you doing for the community, keep up the good work...

**Comment by soakes** [comment]
> > We've found quite a bit of info in our internal chats about this (going through my core dumps and logs).
> > The short version is, right now we're pretty sure the issue has to do with transcoding jobs and updating the transcode status. This happens very frequently, but there is a database query in there which is very heavy and after enough time the backlog locks everything up. That TP Worker is the process running the queries.
> > We're working on a solution so please remain patient and we'll hopefully get that out in a 10.9.2 sometime very soon.
> 
> I can't speak for others but I have to say thank you! My team really appreciate what you doing for the community, keep up the good work...

Totally agree and good job everyone whos contrubuted in anyway as everything helps.

**Comment by VigibotDev** [comment]
> We've found quite a bit of info in our internal chats about this (going through my core dumps and logs).
> 
> The short version is, right now we're pretty sure the issue has to do with transcoding jobs and updating the transcode status. This happens very frequently, but there is a database query in there which is very heavy and after enough time the backlog locks everything up. That TP Worker is the process running the queries.
> 
> We're working on a solution so please remain patient and we'll hopefully get that out in a 10.9.2 sometime very soon.

Even if we use only audio transcoding (I never transcode video, all already 3-5Mbps AVC) ?

**Comment by cvium** [comment]
Direct Stream = Audio, subtitles or container are unsupported
Remux = Container is unsupported

Direct Play is always just sent as-is.

**Comment by SingingFrog7** [comment]
I am pretty sure I had a few occurrences with Direct Play, but let see how it goes with the fix. I weirdly still didn't had crashed in the last 36h, but I disabled all apps using Jellyfin's API (Jellyseerr, jellystat) so I wonder if it helped?

**Comment by soakes** [comment]
> Direct Stream = Audio, subtitles or container are unsupported Remux = Container is unsupported
> 
> Direct Play is always just sent as-is.

Thank you for the clarification.

**Comment by VigibotDev** [comment]
this is a basic to know if you want to take heavy load on a machine :)

**Comment by DwayneGoddem** [comment]
> We've found quite a bit of info in our internal chats about this (going through my core dumps and logs).
> 
> The short version is, right now we're pretty sure the issue has to do with transcoding jobs and updating the transcode status. This happens very frequently, but there is a database query in there which is very heavy and after enough time the backlog locks everything up. That TP Worker is the process running the queries.
> 
> We're working on a solution so please remain patient and we'll hopefully get that out in a 10.9.2 sometime very soon.

You might be right there.
I can see a lot of this error 

I have seen error for "'C:\ProgramData\Jellyfin\Server\transcoding-temp\transcodes\" with the file is open by another process 
This was in transcoded and direct play logs.

I cleared the logs so I don't have a snip to post

**Comment by arab21dwc** [comment]
I have set transcoding thread count to 1, app is still responsive for 12 hours now, will keep it running like this while waiting for the fix or 10.9.2 thanks @joshuaboniface @gnattu @all who contributed to this project keep it up guys

**Comment by memehammad** [comment]
> Anyone have any information about when this bug came up? Earlier I used unstable 10.9.0 from 2023 so my only option is switch back to unstable because 10.8.x will not work with my database.
> 
> My question is all of this releases contain this bug? https://repo.jellyfin.org/files/server/debian/unstable/

Try version 2024040106.

**Comment by gyilkoszabpehely** [comment]
> > Anyone have any information about when this bug came up? Earlier I used unstable 10.9.0 from 2023 so my only option is switch back to unstable because 10.8.x will not work with my database.
> > My question is all of this releases contain this bug? https://repo.jellyfin.org/files/server/debian/unstable/
> 
> Try version 2024040106 or earlier. The issue was introduced in the version after.

Ok, and where can get this version?

**Comment by memehammad** [comment]
> > > Anyone have any information about when this bug came up? Earlier I used unstable 10.9.0 from 2023 so my only option is switch back to unstable because 10.8.x will not work with my database.
> > > My question is all of this releases contain this bug? https://repo.jellyfin.org/files/server/debian/unstable/
> > 
> > 
> > Try version 2024040106 or earlier. The issue was introduced in the version after.
> 
> Ok, and where can I get this version?

No clue 💀💀💀

**Comment by joshuaboniface** [comment]
It would have to be built manually, our CI doesn't keep unstables that old.

**Comment by DwayneGoddem** [comment]
Yesterday I shutdown Jellyseerr and last night with about 6 users streaming both Transcoded and Direct local network and external there was NO issues as all.

so @SingingFrog7 was onto something

So how much was changed in the API engine that could be the issue? or does Jellyseerr just need to update to there end.

Does anyone else use other API based connections?

**Comment by jannik-schmidt** [comment]
> Yesterday I shutdown Jellyseerr and last night with about 6 users streaming both Transcoded and Direct local network and external there was NO issues as all.
> 
> 
> 
> so @SingingFrog7 was onto something
> 
> 
> 
> So how much was changed in the API engine that could be the issue? or does Jellyseerr just need to update to there end.
> 
> 
> 
> Does anyone else use other API based connections? 

I was about to shutdown the whole arr-suite + jellyseerr (every app that uses the API) when Jellyfin started to lock up again.

The moment the apps were down, Jellyfin worked again. 

I'll come back tomorrow and tell if there were issues again.

**Comment by gianklug** [comment]
Using Jellyseerr and other -arr apps that use the API as well. I'll just Jellyseerr down for now and will update if that fixes it.

**Comment by DwayneGoddem** [comment]
> > Yesterday I shutdown Jellyseerr and last night with about 6 users streaming both Transcoded and Direct local network and external there was NO issues as all.
> > so @SingingFrog7 was onto something
> > So how much was changed in the API engine that could be the issue? or does Jellyseerr just need to update to there end.
> > Does anyone else use other API based connections?
> 
> I was about to shutdown the whole arr-suite + jellyseerr (every app that uses the API) when Jellyfin started to lock up again.
> 
> The moment the apps were down, Jellyfin worked again.
> 
> I'll come back tomorrow and tell if there were issues again.

I have all the other err's running with no issues. Just Jellyseerr was stopped and last night was the 1st stable night after upgrading to 10.9.1

**Comment by vin86** [comment]
i can confirm that after disabled jellyseer, jellyfin works without problems

**Comment by DwayneGoddem** [comment]
> i can confirm that after disabled jellyseer, jellyfin works without problems

so I wonder if Jellyfin has an API issue I can see in the change log there was API changes or if Jellyseer needs to be updated?

We will need the devs to check I guess.

So this removes the its a transcode issue i guess.

**Comment by BonzTM** [comment]
Jellyfin 10.9.1 and Jellyseerr 1.8.1 -- no issues here since the release of 10.9.1

**Comment by vincejv** [comment]
@DwayneGoddem jellyserr doesn't need fixing, it's a Jellyfin issue https://github.com/jellyfin/jellyfin/pull/11670

**Comment by felix920506** [comment]
Fixed in #11670

**Comment by SingingFrog7** [comment]
I had re-enabled Jellyseer after updating it and had no issue for another 24h, but then got another crash when a user needed transcoding. Disabling jellyseerr again made it work so I guess it's not just one or the other, but both combined along with other parameter such as a specific virtualization setup (VMs seems fine while LXC have that issue)

I think jellyseerr just add another level of authentication queries which, along with transcoding (which needs to validate if the user is allowed to transcode) ends up being too much. I hope the rolleback to using cache with UserManager will fix it

**Comment by DwayneGoddem** [comment]
yes I guess we need to wait for the next release or try the next unstable release.

the current unstable is dated the 9th so I guess it will just have to keep an eye out for it.

**Comment by felix920506** [comment]
Please test on the next unstable build (should be out next Monday) and open a new issue if it wasn't fixed

**Comment by memehammad** [comment]
> Please test on the next unstable build (should be out next Monday) and open a new issue if it wasn't fixed

Anything stopping you guys from releasing this sooner? Jellyfin is unusable for a lot of people right now.

**Comment by nielsvanvelzen** [comment]
10.9.2 will be released shortly

**Comment by arab21dwc** [comment]
Hi @nielsvanvelzen is it still going to be released today

**Comment by nielsvanvelzen** [comment]
Can't give exact release dates but it should be within 48 hours from now.

**Comment by gyilkoszabpehely** [comment]
If hardware encoding not work and you have Nvidia:
We have reports now of nvidia drivers failure to build on debian’s linux-image-5.10.0-29 recently provided by debian updates (May 7). This feels very similar to what happened back with 6.1.0-15 on bookworm.

My current suggestion for now is to boot with the previous kernel 5.10.0-28 kernel, then to remove the linux-image-5.10.0-29-amd64 kernel and linux-headers-5.10.0-29-amd64, and then purge and reinstall nvidia drivers. removing the kernel will also remove the updating metapackages.

The drivers get partially removed when the attempt to build against the new kernel happens, so reinstall of the nvidia drivers AFTER removing the kernel is necessary.

**Comment by arab21dwc** [comment]
@gyilkoszabpehely problem with this release it crashes the android tv app

**Comment by gyilkoszabpehely** [comment]
> @gyilkoszabpehely problem with this release it crashes the android tv app

I use this release very long time ago, and I tested on all platform...
Are you using hardware encoding? If yes, what kind of video card have you?

**Comment by gyilkoszabpehely** [comment]
I know that most people do not know how useful such a system is if a community or family uses every day.

I have a little dislike for the developers for publishing this edition, maybe they didn't think about how many people's work can go to waste. I know that we need backup, but neither the family nor the community can raise the necessary costs for me.

I could write a novel about my last week, because they destroyed everything that drives the engine of the community for me.
Jellyfin kills jellyfin.
Debian kills Nvidia.

Not funny :(

**Comment by cvium** [comment]
If you can't afford to backup your config and data (not media) directories, I don't think self-hosting is for you.

**Comment by gyilkoszabpehely** [comment]
@cvium if you had bothered to read it back, my life would have been easier :)

**Comment by gyilkoszabpehely** [comment]
> If you can't afford to backup your config and data (not media) directories, I don't think self-hosting is for you.

@cvium I created a backup from everything, except a media. But how can I go back to the previous jellyfin version if I've been using 10.9.0 for almost an year and jellyfin deletes all working version?

**Comment by arab21dwc** [comment]
FIY i also found a way to disable transcoding completely and just uses remuxing for all clients for media and live tv >xui one>xteve>jellyfin and remuxing does not cause high cpu usage

**Comment by SingingFrog7** [comment]
@gyilkoszabpehely you can't go back to 10.8.x unfortunately, due to change in databases. You decided to use unstable releases, that comes with that kind if risks

> I could write a novel about my last week, because they destroyed everything that drives the engine of the community for me.
Jellyfin kills jellyfin.

In that specific case, were are pretty lucky, devs are really responsive and the problem is most likely already fixed (we're just waiting for the next release, but you could build the code yourself if you don't want to wait)

As with most projects like Jellyfin, people aren't doing that for a living, they are maintaining the projects on their own time. 

I don't want to be harsh or anything, but I feel like I'd share a simple "trick" I developed over the year:
- If you don't have the time to troubleshoot/debug, don't use unstable releases and don't upgrade as soon as a new stable release comes out. Wait for a few weeks so that people that have the time/willingness to try them first and troubleshoot can sort issues out with the developers
- Don't share your selfhosted services with friends/family unless you agree to take the time to support them (especially the non-techy ones) and you understand you become responsible, especially if you host their data

**Comment by joshuaboniface** [comment]
To be clear, we don't *delete* anything, so I'm not sure about what is meant there. We don't keep old unstable binary builds sure (ever increasing space requirements), but the code is all there in the Git repository and they can be built again from source if required. Upgrades apply database migrations for code changes however, so yes unless a backup is made before upgrading, upon the very first start of a new version, the on-disk databases will change and will **not** be compatible with any older version(s) - including older unstables potentially.

I'll echo what @SingingFrog7 said. To be exceedingly blunt, if you decide to jump right on a new major release that is 2+ years in the making right on day one... you're going to have problems. We had, at least among the team, at least 15 of us testing 10.9.0 for weeks before releasing and while we noticed (and fixed) a lot of bugs, our usecases cannot collectively represent the literally tens of thousands of other usecases out there. Bugs *will* happen, things *will* break. This is why we constantly stress to take backups before every upgrade, why we ask the community to help test unstable releases (because then we're more likely to find the edge cases, and why we never recommend automatic updates for Jellyfin. If stability is important to your usecase, it is always better to wait at least a week or two before upgrading major versions (note that point releases are much more strict and are usually safe - and recommended for security - to update immediately).

I think at this point, this thread has been exhausted of viable discussion. While I'm normally loathe to do so, I'm going to limit this to contributors only now. 10.9.2, with what we expect is the final fix to this issue, will be released at some point in the next 24-48 hours. If the issue still occurs for anyone after that version, then please open a new issue.
